File: testresources/java/accuracy/src/domain/indirect/violatingfrom/AccessObjectReferenceIndirect_AsReturnValueOfSuperClassMethod_FromSide.java
package domain.indirect.violatingfrom;

import domain.indirect.BaseIndirect;

public class AccessObjectReferenceIndirect_AsReturnValueOfSuperClassMethod_FromSide extends BaseIndirect {
	
	public String initializeProfileInformation(){
		String profile = "profiled";
		Object o = getProfileInformation("tsja", 3);
		return (String) o;
	}
}

File: testresources/csharp/benchmark/Domain/Indirect/ViolatingFrom/AccessObjectReferenceIndirect_AsReturnValueOfSuperClassMethod_FromSide.cs
using Domain.Indirect;

namespace Domain.Indirect.ViolatingFrom
{

	public class AccessObjectReferenceIndirect_AsReturnValueOfSuperClassMethod_FromSide : BaseIndirect

	{
	public string initializeProfileInformation() {
		String profile = "profiled";
		Object o = getProfileInformation("tsja", 3);
		return o;
	}

	}
}
